### ISSUE TITLE
[Bug]: Custom Detection Rules Config URL not Working

### ISSUE THREAD

**Issue by JRM-SC** [CyberDrain/Check#87]
### Required confirmations before submitting

- [x] **I can reproduce this issue on the latest released version** of Check.
- [x] **I have searched existing issues** (both open and closed) to avoid duplicates.
- [x] I am **not** requesting general support; this is an actual bug report.

### Issue Description

I have been trying to get the Check extension to load my custom detection-rules.json which lowers the severity of "phi_014_devtools_blocking" due to false positives across several sites.

Setting a custom detection rule is not working either in the extension UI or via registry keys.  In the UI I set the Config URL, click **Save Settings** and if I refresh the extension settings it's back to the default https://raw.githubusercontent.com/CyberDrain/Check/refs/heads/main/rules/detection-rules.json  ([Screen Capture](https://drive.google.com/file/d/118_yT9DS18IZ-shBjm07NotxQCcX7g18/view).  Clicking **Update Rules Now** with the new config URL also does not update the configuration.

Setting the customRulesUrl registry key in `HKLM:\SOFTWARE\Policies\Google\Chrome\3rdparty\extensions\benimdeioplgkhanklclahllklceahbe\policy` fails to load the custom configuration.  To test I the Update Interval to 1 hour and waited 3 hours to test again but the old detection rules were still in place.

### Extension Version

1.0.5

### Rules Version

1.0.6

### Relevant Logs / Stack Trace

```plaintext

```
